### DISCUSSION TITLE
Build from source without DarkMode?

### DISCUSSION THREAD

**Discussion by ivysrono** [doublecmd/doublecmd#1245] [General]
Build from master and v1.1.1 source by `build.bat release`:
<img width="386" alt="master" src="https://github.com/doublecmd/doublecmd/assets/2649544/79532a77-fa1a-41a7-a4d5-70ca827bf0ae">

Release v1.1.1:
<img width="254" alt="1 1 1" src="https://github.com/doublecmd/doublecmd/assets/2649544/9f7552ca-b444-46a6-8265-f9f9bb346285">

**Comment by alexx2000** [comment]
`build.bat darkwin`

**Reply by ivysrono** [reply]
Thanks！

**Comment by ivysrono** [comment]
@alexx2000 
How to build with `Revision` and `Commit`？
I get source from github release.

```
Double Commander
Version: 1.1.4 beta
Revision: Unknown
Commit: Unknown
Build date: 2023/10/14
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 22H2 x86_64
```

**Comment by alexx2000** [comment]
Simply build from archive `doublecmd-1.1.4-src.tar.gz`.

**Reply by ivysrono** [reply]
Thanks!
